src/lib/learn-client/types.ts

```diff
@@ -188,6 +243,16 @@ export enum HandsOnLabProviderOption {
   instruqt = 'instruqt',
 }
 
+export enum CompanyLogoOption {
```

Comment:
Even though this is currently specific to `ProductPage` data, the miscellaneous enums section felt like a good home.